CCS/2. Concepts/2. 記憶體容量換算 (GB vs. GiB).md

```diff
@@ -1 +1,79 @@
+---
+title: 記憶體容量換算 (GB vs. GiB) | en
+tags: Concept, CCS, EN
+GA: UA-155999456-1
+---
 
+<style>
+#doc.markdown-body, .ui-infobar, .container-thiner {
+  max-width: 1080px;
+}
+.ui-content #doc.markdown-body, .ui-content .ui-infobar {
+    max-width: 900px;
+}
+@media (min-width: 768px) {
+    #doc.markdown-body, .ui-infobar {
+        max-width: 750px;
+    }
+}
+@media (min-width: 1200px) {
+    #doc.markdown-body, .ui-infobar {
+        max-width: 1170px;
+    }
+}
+</style>
+
+{%hackmd @docsharedstyle/default %}
+
+# Memory capacity conversion (GB vs. GiB)
+
+The capacity calculation unit used by "**Memory**" and "**Available memory**"<sup>[1]</sup> is different for Container Computer Service and High-performance Computing. Please refer to this article for detailed differences and conversion formulas.
```

Comment:
[SUG]

The capacity calculation unit used by "**Memory**" and "**Available memory**"<sup>[1]</sup> is different for Container Computer Service and High-performance Computing. > 

The capacity calculation unit- "**Memory**" and "**Available memory**"<sup>[1]</sup>  used in Computer Service and High-performance Computing is different.